usr.sbin/bhyve/vmgenc.c

```diff
@@ -90,7 +90,7 @@ vmgenc_write_dsdt(void)
 	dsdt_line("Name (_CID, \"VM_Gen_Counter\")");
 	dsdt_line("Method (_HID, 0, NotSerialized)");
 	dsdt_line("{");
-	dsdt_line("  Return (\"Bhyve_V_Gen_Counter_V1\")");
+	dsdt_line("  Return (\"bhyve_V_Gen_Counter_V1\")");
```

Comment:
If I read the ACPI specs correctly, this is an invalid _HID value. See https://uefi.org/htmlspecs/ACPI_Spec_6_4_html/06_Device_Configuration/Device_Configuration.html#hid-hardware-id

I don't know who is familiar with ACPI and can help out here.